providers/dns/joker/provider_dmapi.go

```diff
@@ -26,7 +26,7 @@ func newDmapiProvider() (*dmapiProvider, error) {
 		var errU error
 		values, errU = env.Get(EnvUsername, EnvPassword)
 		if errU != nil {
-			// nolint:errorlint
+			// nolint:errorlint,errorlint // false-positive
```

Comment:
Is this a new syntax for the linter (i.e. `//nolint:$name,$name`) or do you *really* want to disable `errorlint` here?

Reply:
just a mistake: :smile: